### PR TITLE
Added Necessary Fixes

### PR DESCRIPTION
**Pull request by Divya063**
Fixes Added:
- Added a validation function for the configuration file 'tests.yaml'.
- Removed exponential notation for floats.
For the output log, I'm creating a separate log for each test i.e, checksum.log for checksum test, throughput.log for throughput test or should I just a create a single log file?
Another pull request is on the way. 
PS- Ignore other branches, I was trying to cherry-pick the commits.

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -27,6 +27,46 @@ def get_config(cfg):
...
+        test_file_path = os.path.join(os.path.join(directory_path, directory), test_file)
+        if not os.path.exists(test_file_path):
+            raise Exception( test_name + " test file does not exists")
+
+def check_input_validity(params):
+    """
+    Check the parameter type
+    """
+    for key , value in params.items():
+        if "number" in key.lower():
```
Comment (ebocchi) [inline run.py]: This is not a very strong assumption (what if I have a `numberOfBytes: 1024`?) but I have no better ideas for the moment...

File: run.py [hunk 2]
```diff
@@ -42,13 +42,15 @@ def check_input_validity(params):
     """
     Check the parameter type
     """
-    for key , value in params.items():
-        if "number" in key.lower():
+    string_val = ['filePath', 'fileSize','repoName', 'repoSize']
```
Comment (ebocchi) [inline run.py]: Better. Feel free to drop my previous comment

**Comment by ebocchi** [comment]
Please, write some quick instructions on how to run the framework!
I really want to start using this machinery

**Comment by Divya063** [comment]
@ebocchi  Sorry for the delay, Currently I'm trying to find a workaround to run the eos test suite from user container, After cloning the framework, I can directly execute "python3 run.py" to run the tests, but then I would not have permission to write the files to eos/user/u/user2/ but after executing the command 'sudo su user2` I'm not able to use python3 command.

File: test.yaml [hunk 3]
```diff
@@ -1,17 +1,22 @@
-tests: 
+# Add the new parameters [here](tests/)
```
Comment (Divya063) [inline test.yaml]: Maybe we can add a comment here along with the link to the line where the parameters have to be added.

**Pull request merged**